README.md

```diff
@@ -1,182 +1,56 @@
-6. Search for "WSL" or "Windows Subsystem for Linux" in the search bar.
-
-7. Select the desired Linux distribution (e.g., Ubuntu, Debian, Fedora) from the search results.
-
-8. Click on the distribution and then click the **Install** button.
-
-9. Wait for the installation to complete. The installation process will download the Linux distribution package from the Microsoft Store.
-
-10. Once the installation is finished, you can launch the Linux distribution from the Start menu or by running its command (e.g., `ubuntu`).
-
-11. The first time you launch the Linux distribution, it will take some time to set up. Follow the on-screen instructions to create a username and password.
-
-12. After the setup is complete, you can use the Linux distribution through WSL on your Windows system.
-
-    
-### Download the input data in "ngen-data" folder from S3 bucket :
-
-#### Linux & Mac & WSL
-
-```Linux   
-    $ mkdir NextGen
-    $ cd NextGen
-    $ mkdir ngen-data
-    $ cd ngen-data
-    $ wget --no-parent https://ciroh-ua-ngen-data.s3.us-east-2.amazonaws.com/AWI-001/AWI_03W_113060_001.tar.gz
-    $ tar -xf AWI_03W_113060_001.tar.gz 
-    $ cd AWI_03W_113060_001
-```
-
-
-#### Windows Steps:
-#### Note: It is recommended to use WSL and follow [instructions for Linux & Mac & WSL](#Linux-&-Mac-&-WSL-)
-
-```Windows  
-    $ mkdir NextGen
-    $ cd NextGen
-    $ mkdir ngen-data
-    $ cd ngen-data
-    $ Invoke-WebRequest -Uri "https://ciroh-ua-ngen-data.s3.us-east-2.amazonaws.com/AWI-001/AWI_03W_113060_001.tar.gz"
-    $ tar -xzf "\AWI_03W_113060_001.tar.gz"
-    $ cd AWI_03W_113060_001
-```
-
-## Run NextGen In A Box
-
-### Clone CloudInfra repo
-
-Navigate to NextGen directory and clone the repo using below commands:
-
-```
-$ git clone https://github.com/CIROH-UA/CloudInfra.git
-
-$ cd CloudInfra
-```  
-Once you are in *CloudInfra* directory, you should see `guide.sh` in it. Now, we are ready to run the model using that script. 
-
-### How to run the model script?
-
-#### WSL, Linux and Mac Steps:
-Follow below steps to run `guide.sh` script 
-```
-    # Note: Make sure you are in ~/Documents/NextGen/CloudInfra directory
-    $ ./guide.sh   
-    
-```
-### Output of the model guide script
-
->*What you will see when you run above `guide.sh`?*
-
-- The script prompts the user to enter the file path for the input data directory where the forcing and config files are stored. 
-
-Run the following command based on your OS and copy the path value:
-
- **Windows:**
-```
-C:> cd ~\<path>\NextGen\ngen-data
-c:> pwd
-and copy the path
+###Reproducing
+
+Get the files from the appropriate bucket & RUN_CONFIG
+```
+wget --no-parent https://awi-ciroh-ngen-data.s3.us-east-2.amazonaws.com/AWI_001/AWI_03W_113060_001.tar.gz .
+
+tar -xvf AWI_03W_113060_001.tar.gz
+```
+Then we can confirm file location and integrity against the example JSON file.
+```
+{
+	"RUN_CONFIG": {
+		"description": "Run Config for AWI_03W_113060_001",
+		"Model": {
+			"realization": {
```

Comment:
Thanks for getting this going, @ZacharyWills! Having now stared at this for 30 minutes and gone back and forth about what comments I want to make, it seems clear to me that we could solve this problem in a lot of different ways. But, as will all solutions, they all have pros and cons. My biggest take away so far is that this initial proposal should set up future proposals for success. I think the way we do that is by going with the simplest data model that does not limit future additions in places where _we want expansion_. 

To get to that place, I think we first need some guard rails on what the spec is _not_. I figure we wont be able to determine all the things it _is_ to start, without using it, so it is likely easier to say what it isn't. So kind of like goals, but anti-goals if you will haha. The purpose of saying what the spec _isn't_ is to _create consensus_ and _avoid scope creep_ and have something to point to when suggestions are brought to the table. Please disagree and challenge me on this. It may just be an idea in my head and not a good one!

My general thoughts so far are (numbered, so it is easier to respond. Not ordered):

1. I think we probably want to create a key within `Model` that is either `files`, `assets`, `configuration` (or something) that is a `list` of realization and configuration record types. This will allow specifying the realization file and all other BMI init config / other supporting files and their details.
2. I like the distinction of configuration, forcing, and hydrofabric. However, im not sold on `realization` needing its own category. To me, its just another configuration file. I think we could instead just have a configuration record type that has `Name: str`, `Store: "fs" | "os"` (this could be confusing, "filesystem" or "bucket" is probably fine), `Path: str`, `Hash: str`, `Variant: "unknown" | "realization" | "cfe" | ...`. This way we can treat all configuration files the same.
3. Im not sold on `Hydrofabric` living within `Forcing`. As you mention in `notes.md`:
  > Firstly there must be a heirarchical underestanding of how we represent data and configurations...

  I don't see it the same. I think you can make the case that you can have forcing over a given hydrofabric domain, so `Forcing` should live within `Hydrofabric`. I am just not sold that the relationship between the two needs hierarchy. Why not just have `Hydrofabric` and `Forcing` as top level keys? My same comment about having `files` or `assets` as a subkey of `Forcings` and `Hydrofabric` applies here too.
4. Where do files like stream flow nudging or restart fall into this?
5. Pedantic: I am not a huge fan of `RUN_CONFIG`. We don't use upper case snake case anywhere else, so I think `RunConfig` or `run_config` are preferred. Even just `Config` or another single word key would be "nicer." Thoughts?
6. Pedantic: use the same case format across keys (i.e. `model` and `realization`, not `Model` and `realization`). My personal preference is lowercase snake case (i.e. `run_config`), but I definitely have a bias from my python background lol. Point being, we should pick something and stick with it.
7. I mention this in a later comment in `notes.md`, but we probaly want to introduce a top level `Metadata` or `Meta` as a top level key and add a `hash` key with its record containing `Algo` or `Type`. So, in json, something like:

  ```json
  {
    "Meta": {
      "Hash": {
        "Algo": "sha256"
      }
    }
  }
  ```

Reply:
So the hierarchy is based off of the assumption that things within the workflow are somewhat dependent (generating forcings for only your selected hydrofabric is more efficient than always generating them everywhere) but there's a lot of discussion about that going on now here: https://github.com/LynkerIntel/hfsubset/issues/28

I totally agree on the syntax changes and I'll put some work in to fix the proposal to reflect those insights. For the most part I just wanted something to get the ball rolling on how `AWI/CIROH-members/OWP//Ops` will work with these run artifacts and the relationship between their configs and the desired outputs. That's ultimately the goal of the metadata is to facilitate that discussion at first, then later on when we're more practiced we can start to talk about how to transition things.

Thanks again for taking a look at this and any input is so valuable at this point to get us going from 0 -> something.